rbd/rbd.go

```diff
@@ -443,47 +447,75 @@ func (image *Image) Rename(destname string) error {
 	return err
 }
 
-// int rbd_open(rados_ioctx_t io, const char *name, rbd_image_t *image, const char *snap_name);
-// int rbd_open_read_only(rados_ioctx_t io, const char *name, rbd_image_t *image,
-//                const char *snap_name);
+// Open the rbd image (DEPRECATED).
```

Comment:
It seems there is some kind of standard/recommendation to mark things deprecated: https://github.com/golang/go/wiki/Deprecated

Reply:
Excellent find! I think the current docs for godoc are not comprehensive/centralized so I had not seen that before. I will update the patch to match that style.